scripts/update/Updater.php

```diff
@@ -245,5 +245,11 @@ public function update($initialVersion)
             $this->setVersion('8.1.0');
         }
 
+        if ($this->isVersion('8.1.0')) {
+            $script = 'sudo -u www-data php index.php \'oat\taoProctoring\scripts\tools\KvMonitoringMigration\' -f context_id -d 1 -s 0 -pc';
```

Comment:
When the data migration script is executed, if the columns are not present in `delivery_monitoring` table then they're created with a default type of `text`.
https://github.com/oat-sa/extension-tao-proctoring/blob/f3934acb851fe1d047fbcdf03a33801a058ac567/scripts/tools/KvMonitoringMigration.php#L206
Columns should be created before the data migration.